### PR TITLE
Some house keeping after -00 publication

### PR DESCRIPTION
**Pull request by bc-pi**

### CODE REVIEW

File: identity-chaining-draft.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ submissiontype: IETF
 area: sec
 wg: oauth
 
-docname: draft-ietf-oauth-identity-chaining-00
+docname: draft-ietf-oauth-identity-chaining-01
```
Comment (aaronpk) [inline identity-chaining-draft.md:9]: this should actually be `-latest` and the build tool will do the right thing

**Review by aaronpk — CHANGES_REQUESTED** [review]

File: identity-chaining-draft.md (hunk 1 above)
Comment (bc-pi) [inline identity-chaining-draft.md:9]:
```suggestion
docname: draft-ietf-oauth-identity-chaining-latest
```

Comment (bc-pi) [reply identity-chaining-draft.md:9]: right, thanks. you'd think i'd be able to remember that...

Comment (bc-pi) [reply identity-chaining-draft.md]: actually, this PR predates the move to that build tool and should probably just be closed with a new PR to fix the id nits non-asciii per 82a4600efce97500f5b128e46e7c0e6a42ab0b4d

**Pull request closed without merging**